pandas/core/indexes/base.py

```diff
@@ -4371,7 +4371,11 @@ def asof_locs(self, where, mask):
         return result
 
     def sort_values(
-        self, return_indexer=False, ascending=True, key: Optional[Callable] = None
+        self,
+        return_indexer=False,
+        ascending=True,
+        key: Optional[Callable] = None,
```

Comment:
can you order these the same as in series (na_position before key)

Reply:
Done.